.devcontainer/base-dev/base-dev.Dockerfile

```diff
@@ -377,10 +369,13 @@ ENV DEBIAN_FRONTEND=
 
 # install dev python3 dependencies
 RUN pip3 install \
-    # to be able to run juypter notebooks
+    # for juypter notebooks
     ipykernel \
     # for integration_tests package
-    types-PyYAML
+    types-PyYAML \
```

Comment:
Can migrate this to its package.xml: https://github.com/UBCSailbot/sailbot_workspace/blob/user/SPDonaghy/migrate-package-specific-dependencies-from-image-to-packages-242/src/integration_tests/package.xml